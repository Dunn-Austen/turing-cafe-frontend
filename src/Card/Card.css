.card {
  background-color: #C4B5B0;
  border: 2px solid #BD8B58;
  color: #018080;
  font-size: 16px;
  font-weight: bold;
  height: 175px;
  margin: 11px;
  width: 140px;
}
