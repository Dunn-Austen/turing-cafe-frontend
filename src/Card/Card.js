import React from 'react';
import './Card.css';

const Card = (props) => {
  return (
    <div className='card'>
      <p>{props.name}</p>
      <p>{props.date}</p>
      <p>{props.time}</p>
      <p>{props.number}</p>
    </div>
  )
}

export default Card;
