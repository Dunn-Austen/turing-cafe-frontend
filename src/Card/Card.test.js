import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card';

describe('card', () => {
  it('snapshot should depict accurately', () => {
    const wrapper = shallow(<Card
      name="Christie"
      date="12/29"
      time="7:00"
      number="12"
      key="Christie12"
    />);

    expect(wrapper).toMatchSnapshot();
  });
});
