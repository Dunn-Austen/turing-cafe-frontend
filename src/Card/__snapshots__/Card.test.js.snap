// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card snapshot should depict accurately 1`] = `
<div
  className="card"
>
  <p>
    Name: 
    Christie
  </p>
  <p>
    Date: 
    12/29
  </p>
  <p>
    Time: 
    7:00
  </p>
  <p>
    Guests: 
    12
  </p>
</div>
`;

exports[`card snapshot should depict accurately 1`] = `
<div
  className="card"
>
  <p>
    Name: 
    Christie
  </p>
  <p>
    Date: 
    12/29
  </p>
  <p>
    Time: 
    7:00
  </p>
  <p>
    Guests: 
    12
  </p>
</div>
`;
